02-create.md

```diff
@@ -73,4 +73,4 @@ nothing to commit (create/copy files and use "git add" to track)
 > git init     # make the beta sub-directory a Git repository
 > ~~~
 > 
-> Why is it a bad idea to do this?
+While it is possible to create one git repository inside another - there is nothing to stop your doing that - this is not a good idea. Can you explain why? 
```

Comment:
Fixed
Belinda


From: Daisie Huang [mailto:notifications@github.com]
Sent: Tuesday, 28 April 2015 6:16 AM
To: swcarpentry/git-novice
Cc: Belinda Weaver
Subject: Re: [git-novice] Adding change directory explanation (#100)


In 02-create.md<https://github.com/swcarpentry/git-novice/pull/100#discussion_r29186083>:

> @@ -73,4 +73,4 @@ nothing to commit (create/copy files and use "git add" to track)

>  > git init     # make the beta sub-directory a Git repository

>  > ~~~

>  >

> -> Why is it a bad idea to do this?

> +While it is possible to create one git repository inside another - there is nothing to stop your doing that - this is not a good idea. Can you explain why?

Can you change these hyphens to em-dashes?

—
Reply to this email directly or view it on GitHub<https://github.com/swcarpentry/git-novice/pull/100/files#r29186083>.